### PR TITLE
Additional db seeds

### PR DESCRIPTION
**Pull request by obozhinov**

### CODE REVIEW

File: db/seed/beneficiary.seed.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  }
+
+  const coordinatorFromDb = await prisma.coordinator.findFirst()
+  console.log(coordinatorFromDb)
+
+  if (!coordinatorFromDb) {
+    throw new Error('No coordinator')
+  }
+
+  var result= new Array();
```
Comment (kachar) [inline db/seed/beneficiary.seed.ts:35]: Usage of `const` is advised over `var` or `let` to provide immutability. There's no problem adding items to the array as the type of the variable is not being changed.

```suggestion
  const result= []
```

File: db/seed/beneficiary.seed.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { PrismaClient, PersonRelation } from '@prisma/client'
+const prisma = new PrismaClient()
+
+export async function beneficiarySeed() {
+  console.log('Beneficiary seed')
+
+  const persons = await prisma.person.findMany({ where: { newsletter: true } })
```
Comment (kachar) [inline db/seed/beneficiary.seed.ts:7]: What's the correlation between people that are subscribed to newsletter and beneficiary seed?

File: db/seed/person.seed.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { PrismaClient } from '@prisma/client'
+const prisma = new PrismaClient()
+
+export async function personSeed() {
+  console.log('Persons seed')
+
+  const insert = await prisma.person.createMany({
+    data: [
+      { firstName: 'Sofia', lastName: 'Vicheva', email: 'sofia@abv.bg', phone: '+359888910910', company: 'ABC', address: 'Stefan Karadja 5, Sofia', newsletter: true},
```
Comment (kachar) [inline db/seed/person.seed.ts:9]: Let's use emails with suffix `@example.com` so we don't accidentally hit somebody's mailbox :sweat_smile:

File: db/seed/beneficiary.seed.ts (hunk 2 above)
Comment (obozhinov) [reply db/seed/beneficiary.seed.ts:7]: that's true I'll get them all

**Review by kachar — APPROVED** [review]

**Pull request merged**